diffusion/datasets/image_caption.py

```diff
@@ -57,8 +56,8 @@ def __init__(
         caption_drop_prob: float = 0.0,
         microcond_drop_prob: float = 0.0,
         caption_selection: str = 'first',
+        crop: Optional[Callable] = None,
```

Comment:
Add blurb about this in docs

`crop_type (str, optional): Type of crop to perform, either ['square', 'random', 'aspect_ratio']. Default: ``'square'``.`